recipe/meta.yaml

```diff
@@ -164,8 +164,7 @@ outputs:
         - suitesparse
         - zlib
         - fenics-ffc =={{ version }}
-        - hdf5 * {{ mpi_prefix }}_*
```

Comment:
`hdf5` has a `run_export` has includes the MPI variant. We can rely on that one.